src/macaron/errors.py

```diff
@@ -54,3 +54,15 @@ class DuplicateError(MacaronError):
 
 class InvalidHTTPResponseError(MacaronError):
     """Happens when the HTTP response is invalid or unexpected."""
+
+
+class CheckRegistryError(MacaronError):
+    """The Check Registry Error class."""
+
+
+class CheckCircularDependencyError(CheckRegistryError):
+    """This error is raised when there is a circular dependency in the registered checks."""
+
+
+class CheckNotExistError(CheckRegistryError):
+    """This error is raised trying to obtain a un-registered check."""
```

Comment:
```suggestion
    """This error is raised when trying to obtain an un-registered check."""
```

Reply:
Thanks for the comment. However after this [discussion](https://github.com/oracle/macaron/pull/254#discussion_r1487025498) this Exception class has been removed in https://github.com/oracle/macaron/pull/254/commits/0be70cbcb7eba77a441802b447da8f4361050f32